sofl/utils/steam_launcher.py

```
@@ -0,0 +1,228 @@
+            # В Flatpak используем flatpak-spawn
+            env_args = []
+            for key, value in env.items():
+                str_value = str(value) if value is not None else ""
+                if str_value.strip():
+                    env_args.append(f"--env={key}={str_value}")
+
+            full_cmd = ["flatpak-spawn", "--host"] + env_args + cmd_argv
+
+            # Добавляем смену директории
+            if game_dir:
+                full_cmd = ["sh", "-c", f"cd {shlex.quote(str(game_dir))} && exec \"$@\"", "sh"] + full_cmd[1:]
```

Ruff (0.12.2)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)